src/player.cpp

```diff
@@ -514,6 +514,8 @@ player::player() :
     nv_cached = false;
     volume = 0;
 
+    thief = THIEF_UNSET; // Is player willing to steal
```

Comment:
Nearly nobody is going to look into the default constructor for documentation of a member. Put documentation where you declare that member (in the header file).